filament/backend/src/ostream.cpp

```diff
@@ -497,6 +497,17 @@ io::ostream& operator<<(io::ostream& out, MRT const& mrt) {
     return out;
 }
 
+io::ostream& operator<<(io::ostream& stream, ShaderStageFlags stageFlags) {
+    unsigned int v = +stageFlags.vertex + 2u * +stageFlags.fragment;
+    switch (v) {
```

Comment:
Match the `to_string` implementation?
```suggestion
    switch (v & 3u) {
```